readme.adoc

```diff
@@ -131,23 +131,17 @@ class hello implements Runnable {
 [source, java]
 ----
 ///usr/bin/env jbang "$0" "$@" ; exit $?
-//DEPS com.sun.net.httpserver:http:20070405
+//DEPS io.javalin:javalin:6.7.0
+//DEPS org.slf4j:slf4j-simple:2.0.17
 
-import com.sun.net.httpserver.*;
-import java.io.IOException;
-import java.net.InetSocketAddress;
+import io.javalin.Javalin;
 
 class WebServer {
-    public static void main(String[] args) throws IOException {
-        HttpServer server = HttpServer.create(new InetSocketAddress(8000), 0);
-        server.createContext("/", exchange -> {
-            String response = "Hello from JBang!";
-            exchange.sendResponseHeaders(200, response.length());
-            exchange.getResponseBody().write(response.getBytes());
-            exchange.close();
-        });
-        server.start();
-        System.out.println("Server started at http://localhost:8000");
+    public static void main(String[] args) {
+        String m = "Hello from JBang!";
+        Javalin app = Javalin.create(/*config*/)
```

Comment:
no point in assigning it to a variable if you dont want to call start separately.